Testing/VelaptorTests/Content/Caching/SoundCacheTests.cs

```diff
@@ -0,0 +1,367 @@
+
+            // Act
+            var actual = cache.CacheKeys;
+
+            // Assert
+            Assert.Single(actual);
+            Assert.Equal(this.oggSoundFilePath, actual[0]);
+        }
+        #endregion
+
+        #region Method Tests
+        [Theory]
+        [InlineData("")]
+        [InlineData(null)]
+        public void GetItem_WithNullOrEmptySoundFilePath_ThrowsException(string filePath)
+        {
+            // Arrange
+            var cache = CreateCache();
+
+            // Act, & Assert
+            AssertExtensions.ThrowsWithMessage<ArgumentNullException>(() =>
+            {
+                cache.GetItem(filePath);
+            }, "The parameter must not be null or empty. (Parameter 'soundFilePath')");
+        }
+
+        [Fact]
+        public void GetItem_WithUnsupportedFileType_ThrowsException()
+        {
+            // Arrange
+            const string dirPath = @"C:\my-sounds\";
+            const string soundName = "test-sound";
+            const string invalidExtension = ".txt";
+            var soundFilePath = $"{dirPath}{soundName}{invalidExtension}";
+            var exceptionMsg = $"Sound file type '{invalidExtension}' is not supported.";
+            exceptionMsg += $"\nSupported file types are '{OggFileExtension}' and '{Mp3FileExtension}'.";
+
+            this.mockPath.Setup(m => m.GetExtension(soundFilePath)).Returns(invalidExtension);
+
+            var cache = CreateCache();
+
+            // Act & Assert
+            AssertExtensions.ThrowsWithMessage<LoadSoundException>(() =>
+            {
+                cache.GetItem(soundFilePath);
+            }, exceptionMsg);
+        }
+
+        [Fact]
+        public void GetItem_WhenOggFileDoesNotExist_ThrowsException()
+        {
+            // Arrange
+            this.mockPath.Setup(m => m.GetExtension(this.oggSoundFilePath)).Returns(OggFileExtension);
+            this.mockFile.Setup(m => m.Exists(this.oggSoundFilePath)).Returns(false);
+            var cache = CreateCache();
+
+            // Act & Assert
+            AssertExtensions.ThrowsWithMessage<FileNotFoundException>(() =>
+            {
+                cache.GetItem(this.oggSoundFilePath);
+            }, $"The '{OggFileExtension}' sound file does not exist.");
+        }
+
+        [Fact]
+        public void GetItem_WhenMp3FileDoesNotExist_ThrowsException()
+        {
+            // Arrange
+            this.mockPath.Setup(m => m.GetExtension(this.mp3SoundFilePath)).Returns(Mp3FileExtension);
+            this.mockFile.Setup(m => m.Exists(this.mp3SoundFilePath)).Returns(false);
+            var cache = CreateCache();
+
+            // Act & Assert
+            AssertExtensions.ThrowsWithMessage<FileNotFoundException>(() =>
+            {
+                cache.GetItem(this.mp3SoundFilePath);
+            }, $"The '{Mp3FileExtension}' sound file does not exist.");
+        }
+
+        [Fact]
+        public void GetItem_WhenGettingSound_ReturnsSound()
```

Comment:
Add code to this unit test to add 2 items with different names to the cache.
This is to check that the correct item is being returned.